### PR TITLE
Update CodeOwners

### PR DESCRIPTION
**Pull request by RickWinter**

### CODE REVIEW

**Review by vhvb1989 — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -6,30 +6,37 @@
...
 
 ###########
 # SDK
 ###########
 
 # Catch all for non-code project files and unowned files | folders
-*                        @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama
-/sdk/                    @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama
+*                        @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama @larryo
+/sdk/                    @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama @larryo
```
Comment (vhvb1989) [inline .github/CODEOWNERS:17]: Do we have a team that we can use here?  like @Azure/cppTeam instead of letting the list go bigger and bigger

**Review by vhvb1989 — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (RickWinter) [reply .github/CODEOWNERS:17]: We don't currently.  I'll get us one.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -6,30 +6,37 @@
...
+/.github/                @AlexGhiondea @jsquire @rickwinter
 
 ###########
 # SDK
 ###########
 
 # Catch all for non-code project files and unowned files | folders
-*                        @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama
-/sdk/                    @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama
+*                        @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama @larryo
```
Comment (ahsonkhan) [inline .github/CODEOWNERS:16]: I am assuming this is supposed to be, @LarryOsterman 
```suggestion
*                        @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama @LarryOsterman
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -6,30 +6,37 @@
...
 
 # PRLabel: %Azure.Identity
 /sdk/identity/           @antkmsft @schaabs @ahsonkhan @rickwinter @vhvb1989 @gearama
 
+###########
+# Client SDKs
+###########
+
+# PRLabel: %Attestation
+/sdk/attestation/        @larryo
```
Comment (ahsonkhan) [inline .github/CODEOWNERS:33]:
```suggestion
/sdk/attestation/        @LarryOsterman
```

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -6,30 +6,37 @@
...
+
 # PRLabel: %KeyVault
 /sdk/keyvault/           @vhvb1989 @gearama @antkmsft @rickwinter
 
 # PRLabel: %Storage
-/sdk/storage/            @vinjiang @katmsft @Jinming-Hu @antkmsft @rickwinter @vhvb1989 @gearama
+/sdk/storage/            @vinjiang @katmsft @Jinming-Hu @antkmsft @vhvb1989 @gearama @larryo
```
Comment (ahsonkhan) [inline .github/CODEOWNERS:39]: Same here and elsewhere.

```suggestion
/sdk/storage/            @vinjiang @katmsft @Jinming-Hu @antkmsft @vhvb1989 @gearama @LarryOsterman
```

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -6,30 +6,37 @@
...
+*                        @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama @larryo
+/sdk/                    @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama @larryo
 
 # Samples
-/samples/                @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama
+/samples/                @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama @larryo
 
 # PRLabel: %Azure.Core
-/sdk/core/               @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama
+/sdk/core/               @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama @larryo
```
Comment (ahsonkhan) [inline .github/CODEOWNERS:23]:
```suggestion
/sdk/core/               @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama @LarryOsterman
```

File: .github/CODEOWNERS [hunk 6]
```diff
@@ -6,30 +6,37 @@
...
 
 # Catch all for non-code project files and unowned files | folders
-*                        @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama
-/sdk/                    @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama
+*                        @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama @larryo
+/sdk/                    @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama @larryo
 
 # Samples
-/samples/                @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama
+/samples/                @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama @larryo
```
Comment (ahsonkhan) [inline .github/CODEOWNERS:20]:
```suggestion
/samples/                @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama @LarryOsterman
```

File: .github/CODEOWNERS (hunk 1 above)
Comment (ahsonkhan) [inline .github/CODEOWNERS:17]:
```suggestion
/sdk/                    @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama @LarryOsterman
```

**Pull request merged**